layouts/shortcodes/blog.html

```diff
@@ -0,0 +1,41 @@
+{{- $maxBlogPosts := 10 }}
```

Comment:
I guess this is a custom shortcode you created to support the blogs?

Reply:
Yes. Although it is unused. I may remove it if we won't bring back that block.